.github/workflows/gh-pages.yml

```
@@ -360,8 +383,19 @@ jobs:
 
   publish:
     name: Publish
+    if: |
+      github.event_name == 'push' &&
+      github.ref == 'refs/heads/main' &&
```

actionlint (1.7.12)

[error] property "source" is not defined in object type {changes: string}

(expression)